HHS/OPRE-OPS
# feat: ✨ adds AgreementBLIAccordion component

## What changed
Round one
-feat: ✨ adds AgreementBLIAccordion component

Does not include
- table actions
- table filters based on action selection
- after approval toggle

## Issue

#1558 

## How to test

1. go to Agreements list 
2. review agreement 1
3. see BLIAccordion
4. 💰profit

## Screenshots

_If relevant, e.g. for a front-end feature_

## Definition of Done Checklist
